src/Microsoft.DotNet.Interactive.Notebooks/Notebook/NotebookFileFormatHandler.cs

```diff
@@ -26,30 +25,35 @@ static NotebookFileFormatHandler()
             _serializerOptions.Converters.Add(new DataDictionaryConverter());
         }
 
-        public static NotebookDocument Parse(string fileName, byte[] rawData, string defaultLanguage, IDictionary<string, string> kernelLanguageAliases)
+        public static NotebookDocument Parse(string fileName, string notebookContent, string defaultLanguage,
+            IDictionary<string, string> kernelLanguageAliases)
         {
             var extension = Path.GetExtension(fileName);
             switch (extension.ToLowerInvariant())
             {
                 case ".dib":
                 case ".dotnet-interactive":
-                    return ParseInteractiveNotebook(rawData, defaultLanguage, kernelLanguageAliases);
+                    return ParseInteractiveNotebook(notebookContent, defaultLanguage, kernelLanguageAliases);
```

Comment:
Any reason to keep support for the `.dotnet-interactive` file extension?

Reply:
@brettfo ?

Reply:
Could you open an issue for this and we'll revisit?